coverage_map/src/lib.rs

```diff
@@ -382,12 +358,9 @@ mod test {
     }
 
     fn outdoor_cbrs_coverage(cbsd_id: &str, hex: u64, signal_power: i32) -> CoverageObject {
-        let owner: PublicKeyBinary = "112NqN2WWMwtK29PMzRby62fDydBJfsCLkCAf392stdok48ovNT6"
-            .parse()
-            .expect("failed owner parse");
         CoverageObject {
             indoor: false,
-            hotspot_key: owner,
+            hotspot_key: vec![0, 0],
```

Comment:
it's not very important, but these don't even need to be length 2. They could just be `vec![0]`, `vec![1]`, etc.